client/src/app/containers/Home/ActiveGamesTable.jsx

```diff
@@ -0,0 +1,119 @@
+import React from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { useInjectReducer, useInjectSaga } from 'redux-injectors';
+import { selectCurrentUser } from '../../../auth/selectors';
```

Comment:
This is a me thing but we always use absolute path over retaliative. It's clean and you actually know where the imports are located.

Reply:
Do we need to do something special in the config to allow this?